core/src/main/scala/com/databricks/labs/remorph/parsers/intermediate/functions.scala

```diff
@@ -252,7 +249,10 @@ class CallMapper {
     case CallFunction("SOUNDEX", args) => SoundEx(args.head)
     case CallFunction("SPACE", args) => StringSpace(args.head)
     case CallFunction("SPARK_PARTITION_ID", _) => SparkPartitionID()
-    case CallFunction("SPLIT", args) => StringSplit(args.head, args(1), args(2))
+    case CallFunction("SPLIT", args) =>
+      val delim = if (args.size >= 3) Some(args(2)) else None
+      StringSplit(args.head, args(1), delim)
+    case CallFunction("SPLIT_PART", args) => StringSplitPart(args.head, args(1), args(2))
```

Comment:
we don't have `StringSplitPart` in Spark. Convert the call in the optimizer rule.

Reply:
In fact we do https://docs.databricks.com/en/sql/language-manual/functions/split_part.html